cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -309,6 +311,8 @@ $(document).ready(function () {
 
 			// clear interval
 			clearInterval(this.interval);
+
+			$(window).trigger('structureboard_hidden.sideframe');
```

Comment:
@mkoistinen I think that bubbling makes sense for events about DOM elements. However I don't think it makes sense for custom events that don't relate to a particular DOM element. I think they should all be thrown at, and caught on, a specific element for consistency. And for reasons of efficiency stated above (to avoid the overhead of bubbling), I think that should be $(window).